### PR TITLE
Use npm to manage external modules

### PR DESCRIPTION
**Pull request by itrich**
## Current situation
Currently, external modules like Bootstrap, jQuery, etc. are downloaded, extracted and copied manually into their corresponding directories.

## Preferred situation
We have a module manager that automatically updates all external modules and solves dependencies correctly.

## Why?
This PR introduces npm as module manager for external packages. This has several advantages:
- We can update all packages by simply calling `npm update`
- We can make use of [ORT](https://oss-review-toolkit.org/) to audit our `package.json` bill of material
- We can make use of Dependabot to check for known vulnerabilities.

The integration is a very poor man's approach as I have simply symlinked the relevant files to the corresponding package directory. However, introducing yet another tool like [webpack.js](https://webpack.js.org/) is overwhelming and not really needed.

### CODE REVIEW

**Comment by itrich** [comment]
This PR addresses #203

File: _assets/css/MarkerCluster.SCS.css [hunk 1]
```diff
@@ -21,6 +21,10 @@
 	background-color: rgb(138, 212, 193);
 	}
 
+.leaflet-container .leaflet-marker-pane img {
+	width: 34px !important;
+}
+
```
Comment (fkr) [inline _assets/css/MarkerCluster.SCS.css:27]: this seems to be a functional addition?

Comment (fkr) [reply _assets/css/MarkerCluster.SCS.css:27]: nevermind, found the reason later on :)

Comment (itrich) [reply _assets/css/MarkerCluster.SCS.css:27]: No, it's a hotfix since leaflet changed the default width of images in a newer version.

**Review by fkr — APPROVED** [review]
I like the idea of the PR and the way it is approached

**Pull request merged**